### PR TITLE
Pseudo screen

### PR DESCRIPTION
**Pull request by tanguymossion**
simulator_screenshot_A162D14B-382C-49A6-B440-67EE0845F17C

### CODE REVIEW

**Review by Scuym — APPROVED** [review]

File: lib/shared/theme/text_input.dart [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      padding: const EdgeInsets.all(5),
+      decoration: BoxDecoration(
+        color: Colors.white,
+        borderRadius: BorderRadius.circular(20),
+      ),
+      child: TextFormField(
+        onChanged: onChanged,
+        textAlign: TextAlign.center,
+        decoration: const InputDecoration(
+          border: InputBorder.none,
```
Comment (tanguymossion) [inline lib/shared/theme/text_input.dart]: focus on load

File: lib/shared/theme/continue_button.dart [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        padding: const EdgeInsets.symmetric(
+          horizontal: 36,
+          vertical: 10,
+        ),
+        decoration: BoxDecoration(
+          color: Colors.black,
+          borderRadius: BorderRadius.circular(10),
+        ),
+        child: SvgPicture.asset('assets/svg/right_arrow.svg'),
+      ),
```
Comment (tanguymossion) [inline lib/shared/theme/continue_button.dart]: need feedback

Comment (tanguymossion) [reply lib/shared/theme/continue_button.dart]: need to do onpressed when keyboard enter

**Review by Scuym — APPROVED** [review]

**Comment by Scuym** [comment]
Résolue et merged à la main, à delete ?

**Pull request closed without merging**